guides/common/modules/proc_tuning-apache-httpd-child-processes.adoc

```diff
@@ -0,0 +1,31 @@
+[id="tuning_apache_httpd_child_processes_{context}"]
+= Tuning Apache Httpd Child Processes
+
+By default, httpd uses event request handling mechanism.
+When the number of requests to httpd exceeds the maximum number of child processes that can be launched to handle the incoming connections, httpd raises an HTTP 503 Service Unavailable error.
+Amidst httpd running out of processes to handle, the incoming connections can also result in multiple component failures on your {Project} services side due to the dependency of some components on the availability of httpd processes.
+
+You can adapt the configuration of httpd event to handle more concurrent requests based on your expected peak load.
+
+[WARNING]
+====
+Configuring these numbers in `custom-hiera.yaml` locks them.
+If you change these numbers using `{foreman-installer} --tuning=_My_Tuning_Option_`, your `custom-hiera.yaml` will overwrite this setting.
+Set your numbers only if you have specific need for it.
+====
+
+.Procedure
+. Modify the number of concurrent requests in '/etc/foreman-installer/custom-hiera.yaml' by changing or adding the following lines:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+apache::mod::event::serverlimit: 64
+apache::mod::event::maxrequestworkers: 1024
+apache::mod::event::maxrequestsperchild: 4000
+----
++
+The example is identical to running `{foreman-installer} --tuning=medium` or higher on {ProjectServer}.
+. Apply your changes to {ProjectServer}.
+For more information, see xref:Applying_configurations_{context}[].
+
+
```

Comment:
```suggestion
```

Reply:
@Imaanpreet Just this one small improvement and we're ready to merge. Thank you!

Reply:
thank you guys @maximiliankolb and @Lennonka !